spec/01-unit/08-router_spec.lua

```diff
@@ -324,12 +324,14 @@ for _, flavor in ipairs({ "traditional", "traditional_compatible" }) do
         assert.same(nil, match_t.matches.uri_captures)
       end)
 
-      it_trad_only("[host] ignores default port", function()
+      it("[host] ignores default port", function()
         -- host
         local match_t = router:select("GET", "/", "domain-1.org:80")
         assert.truthy(match_t)
         assert.same(use_case[1].route, match_t.route)
-        assert.same(use_case[1].route.hosts[1], match_t.matches.host)
+        if flavor == "traditional" then
+          assert.same(use_case[1].route.hosts[1], match_t.matches.host)
```

Comment:
Why does atc-router has no field `match_t.matches.host` ?

Reply:
This field makes no sense in a DSL especially if `host` metched multiple conditions with `&&` or `
||`, plus this is only used by tests so we choose to not support it.